src/Uno.UWP/Devices/Radios/Radio.Android.cs

```diff
@@ -0,0 +1,252 @@
+
+		private static Radio GetRadiosBluetooth()
+		{
+			Android.Content.Context context = Android.App.Application.Context;
+			bool btFull = context.PackageManager.HasSystemFeature(Android.Content.PM.PackageManager.FeatureBluetooth);
+			bool btLE = context.PackageManager.HasSystemFeature(Android.Content.PM.PackageManager.FeatureBluetoothLe);
+			if (!(btFull || btLE))
+			{
+				return null;
+			}
+
+			var oRadio = new Radio();
+			oRadio.Kind = RadioKind.Bluetooth;
+			oRadio.Name = "Bluetooth";  // name as in UWP
+
+			if (!Windows.Extensions.PermissionsHelper.IsDeclaredInManifest(Android.Manifest.Permission.Bluetooth))
+			{
+				oRadio.State = RadioState.Unknown;
+				return oRadio;
+			}
+
+			if (Android.OS.Build.VERSION.SdkInt <= Android.OS.BuildVersionCodes.JellyBeanMr1)
+			{
+				// deprecated in API 31
+#pragma warning disable CS0618 // Type or member is obsolete
+				var btAdapter = Android.Bluetooth.BluetoothAdapter.DefaultAdapter;
+#pragma warning restore CS0618 // Type or member is obsolete
+				if (btAdapter == null)
+				{
+					return null;    // shouldn't happen...
+				}
+				else if (!btAdapter.IsEnabled)
+				{
+					oRadio.State = RadioState.Off;
+				}
+				else
+				{
+					oRadio.State = RadioState.On;
+				}
+
+				return oRadio;
+			}
+			else
+			{
+				var btAdapter = ((Android.Bluetooth.BluetoothManager)Android.App.Application.Context.GetSystemService(Android.Content.Context.BluetoothService)).Adapter;
+				if (btAdapter == null)
+				{
+					return null;    // shouldn't happen...
+				}
+				if (btAdapter.State != Android.Bluetooth.State.On)
+				{
+					oRadio.State = RadioState.Off;
+				}
+				else
+				{
+					oRadio.State = RadioState.On;
+				}
+
+				return oRadio;
+			}
+
+		}
+
+		private static Radio GetRadiosWiFi()
+		{
+			Android.Content.Context context = Android.App.Application.Context;
+			if (!context.PackageManager.HasSystemFeature(Android.Content.PM.PackageManager.FeatureWifi))
+			{
+				return null;
+			}
+
+			var oRadio = new Radio();
+			oRadio.Kind = RadioKind.WiFi;
+			oRadio.Name = "Wi-Fi";  // name as in UWP
+
+			if (!Windows.Extensions.PermissionsHelper.IsDeclaredInManifest(Android.Manifest.Permission.AccessNetworkState))
+			{
+				oRadio.State = RadioState.Unknown;
+				return oRadio;
+			}
+
+			var connManager = (Android.Net.ConnectivityManager)context.GetSystemService(Android.Content.Context.ConnectivityService);
+
+			if (Android.OS.Build.VERSION.SdkInt > Android.OS.BuildVersionCodes.LollipopMr1)
+			{
+				// since API 23
+				var activeNetwork = connManager.ActiveNetwork;
+				if (activeNetwork is null)
+				{
+					return null;
+				}
+				var netCaps = connManager.GetNetworkCapabilities(activeNetwork);
+				if (netCaps is null)
+				{
+					return null;
+				}
+				if (netCaps.HasTransport(Android.Net.TransportType.Wifi))
+				{
+					oRadio.State = RadioState.On;
+				}
+				else
+				{
+					oRadio.State = RadioState.Off;
+				}

```

Comment:
This block is the same as for cellular. Make a method out of it with the TransportType as a parameter.

Reply:
GetRadioStateForNet created.